extensions/amp-consent/0.1/consent-config.js

```diff
@@ -150,7 +155,37 @@ export class ConsentConfig {
       }
     }
 
-    return config;
+    // (TODO): Migrate from prompIfUnknownForGeo to geoOverride
+    // E.g: promptIfUnknownForGeo: geoGroup -> geoOverride: {geoGroup: {consentRequired: true}}
+
+    // Do I need to guard against incorrect geoOverride?
```

Comment:
let's merge first and then validate

Reply:
new validation rules
- if `consentRequired: "remote"`, then `checkConsentHref` has to be specified.